### PR TITLE
Adding Assistant space

### PR DESCRIPTION
**Pull request by datavistics**
I added a pull request to give a bit more information for blog creators in the readme. I added a huggingface space to show how to do some of the tricky markdown and also mention using the community blog editor for its preview functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,3 +79,15 @@ A couple of alternatives:
...
+
+Feel free to check out this [assistant space](https://huggingface.co/spaces/derek-thomas/hugging-face-blog-assistant) 
+to help with rending some of the tricky markdown like:
+- Inserting caption figures with image tables
+- Tip formatting
+- Inserting clickable references
+ 
+Do Give it a like if it was helpful.
+
+Also you can use the [community blog editor](https://huggingface.co/new-blog) which has a preview function.
```
Comment (julien-c) [inline README.md:93]: maybe invert the order and put this one first

Comment (datavistics) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -79,3 +79,16 @@ A couple of alternatives:
 ## How to Use Highlight Boxes
 
 See an example [in this PR](https://github.com/huggingface/blog/pull/1180). This is to be used sparingly :)
+
+## More Assistance
+
+You can use the [community blog editor](https://huggingface.co/new-blog) which has a preview function.
+
+Feel free to check out this [assistant space](https://huggingface.co/spaces/derek-thomas/hugging-face-blog-assistant)
```
Comment (pcuenca) [inline README.md:87]:
```suggestion
Or check out this [assistant space](https://huggingface.co/spaces/derek-thomas/hugging-face-blog-assistant) 
```

File: README.md [hunk 3]
```diff
@@ -79,3 +79,16 @@ A couple of alternatives:
 ## How to Use Highlight Boxes
 
 See an example [in this PR](https://github.com/huggingface/blog/pull/1180). This is to be used sparingly :)
+
+## More Assistance
+
+You can use the [community blog editor](https://huggingface.co/new-blog) which has a preview function.
+
+Feel free to check out this [assistant space](https://huggingface.co/spaces/derek-thomas/hugging-face-blog-assistant) 
+to help with rending some of the tricky markdown like:
```
Comment (pcuenca) [inline README.md:88]:
```suggestion
to help with rendering some tricky markdown blocks like:
```

**Review by pcuenca — APPROVED** [review]
Nice!

**Pull request merged**